cocos/core/root.ts

```diff
@@ -490,9 +490,7 @@ export class Root {
 
             legacyCC.director.emit(legacyCC.Director.EVENT_BEFORE_COMMIT);
             cameraList.sort((a: Camera, b: Camera) => a.priority - b.priority);
-            if (!SERVER_MODE) {
```

Comment:
Why delete this check? No need anymore?

Reply:
The upstream codes already modified it. I don't know why it also has this modification here. I don't change it.